src/testdrive/src/error.rs

```diff
@@ -178,10 +203,16 @@ impl fmt::Display for Error {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         match self {
             Error::Input { err, .. } => write!(f, "{}", err.msg),
-            Error::General { ctx, cause, .. } => {
+            Error::General { ctx, causes, .. } => {
                 write!(f, "{}", ctx)?;
-                if let Some(cause) = cause {
-                    write!(f, ": {}", cause)?;
+                write!(f, "{}", ctx)?;
```

Comment:
This line looks like a duplicate of the first?

Reply:
weird.